jwql/website/apps/jwql/views.py

```diff
@@ -402,6 +414,209 @@ def not_found(request, *kwargs):
     return render(request, template, context, status=status_code)
 
 
+def query_anomaly(request):
```

Comment:
Great work on these views.  Very nicely constructed!

Reply:
Thank you!